bad.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class Thing_In_A_Box {
```

Comment:
Each class must be in a namespace of at least one level (a top-level vendor name)
Class name "Thing_In_A_Box" is not in camel caps format
Opening brace of a class must be on the line after the definition